src/Client/Invoice.php

```diff
@@ -207,33 +207,6 @@ public function markInvoiceStatus(string $storeId, string $invoiceId, string $ma
         }
     }
 
-    public function updateInvoice(
```

Comment:
Good catch :+1: Interesting as it is not on master, seems to be caused by some merge/rebase, probably when I rebased last time :sweat_smile: